modules/rubiconBidAdapter.js

```diff
@@ -255,11 +255,11 @@ export const spec = {
       }
 
       const bidFpd = {
-        user: bidRequest.params.visitor || {},
-        context: bidRequest.params.inventory || {}
+        user: {...bidRequest.params.visitor} || {},
+        context: {...bidRequest.params.inventory} || {}
```

Comment:
Not sure this will work as expected.

Why did you change it from the previous

```
      const bidFpd = {
        user: bidRequest.params.visitor || {},
        context: bidRequest.params.inventory || {}
      }
```

Reply:
You could just leave it as:

```
{...bidRequest.params.visitor}
{...bidRequest.params.inventory}
```

And it will do what is intended (default to {})

The || {} never will get executed.

Reply:
I think I removed this already no? I pushed twice this morning